display/d.vect/main.c

```diff
@@ -516,7 +516,7 @@ int cmp(const void *a, const void *b)
 char *icon_files(void)
 {
     char **list, *ret;
-    char buf[GNAME_MAX], path[GPATH_MAX], path_i[GPATH_MAX];
+    char buf[GNAME_MAX + GNAME_MAX], path[GPATH_MAX], path_i[GPATH_MAX];
```

Comment:
Same code as in d.linegraph, a path is stored in buf, thus GPATH_MAX seems appropriate.

Reply:
see https://github.com/OSGeo/grass/pull/1406#discussion_r595225478